client/components/product-card/index.jsx

```diff
@@ -59,14 +60,17 @@ const ProductCard = ( {
 				</div>
 			</div>
 			<div className="product-card__description">
+				{ description && <p>{ description }</p> }
 				{ purchase && (
-					<p>
-						<a href={ managePurchase( purchase.domain, purchase.id ) }>
+					<div class="product-card__action">
```

Comment:
I like the approach of using something that exists here.

However 😬

The classes that we used are internal for the `ProductCardAction` component. Can we use that component instead, and just make the `primary` prop an optional one in that component (and optionally pass `href` down to it)? The reason I suggest that is that it allows for us to reuse all of the functionality here instead of building new edge cases.

What do you think?